deploy/generate-deployment.sh

```diff
@@ -105,7 +105,16 @@ for var in "${required_vars[@]}"; do
   fi
 done
 
-required_bin=(kustomize envsubst csplit yq)
+if [ -z ${KUSTOMIZE} ]; then
+  echo "Required env var KUSTOMIZE not set. Set KUSTOMIZE to point to a kustomize v4.0.5 binary"
```

Comment:
Seems it's going to break release script.

Reply:
:disappointed: I should've known!

I'll look into it. I assume we'll need to do something similar during release to avoid the same versioning issue we're facing everywhere else.

Reply:
but don't duplicate. Let's move kustomize install into script or make release uses makefile.